contracts/colony/Colony.sol

```diff
@@ -48,6 +48,27 @@ contract Colony is ColonyStorage, PatriciaTreeProofs, MultiChain {
   function makeArbitraryTransaction(address _to, bytes memory _action)
   public stoppable auth
   returns (bool)
+  {
+    return makeArbitraryTransactionFunctionality(_to, _action);
+  }
+
+  function makeArbitraryTransactions(address[] memory _targets, bytes[] memory _actions)
+  public stoppable auth
+  returns (bool)
+  {
+    require(_targets.length == _actions.length, "colony-targets-and-actions-length-mismatch");
+    for (uint256 i = 0; i < _targets.length; i+= 1){
+      require(
+        makeArbitraryTransactionFunctionality(_targets[i], _actions[i]),
```

Comment:
Why not just call `makeArbitraryTransaction`? Internal calls are authorized by default.

Reply:
Ah, you want to deprecate the single version.

Reply:
My only request is that we rename the internal function `makeArbitraryTransaction` when the time comes.